test/integration/gcp/attributes.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 Google LLC
```

Comment:
This file is listed in the .gitignore, but it's still present - I think it superseded the .gitignore and is still floating around in the repo. Let's delete this file.